data/mods/Dark-Skies-Above/monsters/alien_robots.json

```diff
@@ -50,6 +75,7 @@
     "vision_night": 5,
     "special_attacks": [
       [ "TAZER", 12 ],
+      [ "DSA_DRONE_SCAN", 5 ],
```

Comment:
My only concern with giving the seeker the scan is that it's already a somewhat dangerous attacker when angered. That said, it certainly does make them a much more dangerous enemy, so it's just a consideration.